### ISSUE TITLE
AcmDropdown dropdown items are not keyboard accessible 

### ISSUE THREAD

**Issue by RobDolares** [open-cluster-management/ui-components#186]
The AcmDropdown is keyboard accessible to expand the dropdown item list, but the dropdown items are not accessible.

<img width="419" alt="Screen Shot 2021-01-29 at 4 10 17 PM" src="https://user-images.githubusercontent.com/10394596/106327770-b3e38100-624c-11eb-93ff-71bfba01fb5d.png">
